fix: Update instructions following the Airbyte Cloud integration (#57)
KYVENetwork/docs: docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md

1. Set up the source.

>    Search "KYVE" in the `Sources searchbar` and select it. 

Reviewer: ```suggestion
   Search for "KYVE" in the `Sources search bar` and select it. 
```

>    Search for "KYVE" in the `Sources search bar` and select it.

Reviewer 2: @cgkournelos-rf Convert it to "`Sources` search bar".

   <br></br>
   <img src="/img/elt/airbyte_kyve_source.jpg" alt="Airbyte set up source" />

>    ***NOTE***: The **Bundle-Start-IDs** determines the size of the extracted data. This also affects the time that it is required for completing
>    the sync process which can take place for hours if the data are old. It is advised for testing purposes to visit the [KYVE network Polls page](https://app.korellia.kyve.network/#/pools) and to select a recent bundle ID (2-5 days before).

Reviewer: ```suggestion
   ***NOTE***: The **Bundle-Start-IDs** determines the size of the extracted data. This also affects the time required for completing
   the sync process, up to several hours for historic data. It is advised for testing purposes to visit the [KYVE network Polls page](https://app.korellia.kyve.network/#/pools) and to select a recent bundle ID (2-5 days old).
```

2. Set up the destination.

>    Search for "Postgres" in the `Destinations search bar` and select it.

Reviewer 2: @cgkournelos-rf Convert it to "`Destinations` search bar".

>    Search "Postgres" in the `Destinations searchbar` and select it.

Reviewer: ```suggestion
   Search for "Postgres" in the `Destinations search bar` and select it.
```

3. Final ELT configuration.

   Select the `KYVE Source` that you have configured in step 1 and the `Postgres Destination` from step 2.

>    In this final step  you can modify the pipeline.

Reviewer: ```suggestion
   In this step you can make final modifications to the pipeline.
```
